src/util/permissionIdentities.tsx

```diff
@@ -0,0 +1,183 @@
+import { LxdGroup, LxdIdentity } from "types/permissions";
+
+export type IdentitiesOrGroupsChangeSummary = Record<
```

Comment:
```suggestion
export type ChangeSummary = Record<
```
less precise, but clear enough?

Reply:
Yeah agreed :+1: